libvisual/tools/lv-tool/lv-tool.cpp

```diff
@@ -128,10 +128,14 @@ namespace LV {
             return Time(SDL_GetTicks());
         }
 
-        Time operator-(Time & other) const {
+        Time operator-(Time const& other) const {
             return Time(this->m_moment - other.m_moment);
         }
 
```

Comment:
What was the reason to implement the subtraction operator as a member function? The  implementation in master uses a free function, following the practice of the C++ standard library.

Reply:
@kaixiong no particular reason, changed now as suggested.